modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,61 @@
+The calendar defines the date and time of your maintenance window events, including recurring events, and must be in [path]``ical`` format.
+The schedule uses the events defined in the calendar to create the maintenance windows.
+You must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar, before you can create the schedule.
+
+When you have created the schedule, you can assign it to clients that are registered to the {productname} Server.
+Clients that have a maintenance window assigned cannot run restricted actions outside of the scheduled maintenance window.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Restricted actions include:
+
+* Package installation
+* Client upgrade
+* Service pack migration
+* Highstate application (for Salt clients)
+
+Unrestricted actions are minor actions that are considered safe and are unlikely to cause problems on the client.
+Unrestricted actions include:
+
+* Package profile update
+* Hardware refresh
+* Subscribing to software channels
+
+
+Before you begin, you must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar.
+You can create [path]``ical`` files in your preferred calendaring tool, such as KOrganizer.
+// I don't know if this is correct. ---LKB 2020-07-21
+
+
+
+.Procedure: Creating a New Maintenance Calendar
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Calendar], and click btn:[Create].
+. In the [guimenu]``Calendar Name`` section, type a name for your calendar.
+. Either provide a URL to your [path]``ical`` file, or upload the file directly.
+. Click btn:[Create Schedule] to save your calendar.
+// Well, that;s confusing :\ -- LKB 2020-07-21
+
+
+
+.Procedure: Creating a New Schedule
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Calendar], and click btn:[Create].
+. In the [guimenu]``Schedule Name`` section, type a name for your schedule.
+. OPTIONAL: If your [path]``ical`` file contains more than one calendar, and you want to schedule all of the calendars, check [guimenu]``Multi``.
+. Select the calendar to assign to this schedule.
+. Click btn:[Update Schedule] to save your schedule.
+
+
+
+.Procedure: Assigning a Schedule to a Client
+. In the {productname} {webui}, navigate to menu:Systems[Systems List], select the client to be assigned to a schedule, locate the [guimenu]``Properties`` panel and click btn:[Edit These Properties].
+Alternatively, you can assign clients through the system set manager.
+. In the [guimenu]``Edit System Details`` page, locate the [guimenu]``Maintenance Schedule`` field, and select the name of the schedule to be assigned.
+. Click btn:[Update Properties] to assign the maintenance schedule.
+. OPTIONAL: To remove a maintenance schedule, check the btn:[Cancel affected actions] option.
```

Comment:
The "cancel affected actions" function can be used in situations, when you have some (restricted) actions for a system that are scheduled for the future and you want to assign a schedule to that system. 

Let's consider this example:
- you have a system that is not assigned to any maint. schedules
- you schedule a highstate application to happen in the future, e.g. `2020-12-12 10:00 (Saturday)`
- now you want to assign a maintenance schedule to this system, but the maintenance schedule only contains windows on weekdays
- obviously, the schedule is not compatible with the existing scheduled action (highstate application), so if you check "cancel affected actions", SUMA will cancel the scheduled highstate execution of that highstate upon assigning the schedule
- (otoh, if you leave that unchecked and you assign that schedule, SUMA will report a warning, that assigning that schedule is not allowed)

Therefore I'd rephrase into something like this:
```suggestion
. OPTIONAL: Checking the btn:[Cancel affected actions] cancels already scheduled actions that collide with the maintenance schedule.
```

Reply:
Ah, that makes sense now! I've added this into an admonition below the procedure.